bundles/org.openhab.binding.sonnen/README.md

```diff
@@ -40,6 +40,8 @@ The following channels are yet supported:
 | energyExportedStateProduction  | Number:Energy | read   | Indicates the exported kWh Production                                                   |
 | energyImportedStateConsumption | Number:Energy | read   | Indicates the imported kWh Consumption                                                  |
 | energyExportedStateConsumption | Number:Energy | read   | Indicates the exported kWh Consumption                                                  |
+| batteryChargingFromGrid | Switch| read/write   | Starts and stops the active battery charging from Grid. Note: "Write-API" in Software-Integration page of the local web interface from the sonnen battery must be activated and the given token must be entered in authentication token field of the binding|
```

Comment:
Please use markdowntable.com or your ide markdown function to align this table.

Reply:
ok